### ISSUE TITLE
Better library name

### ISSUE THREAD

**Issue by malukenho** [psr7-sessions/session-encode-decode#1]
Any suggestions?

**Comment by lcobucci** [comment]
`psr7-sessions/encoder` seems to be enough...

**Comment by Ocramius** [comment]
What is the purpose? As in: where would this be used?

**Comment by malukenho** [comment]
@Ocramius did you read the `README`?

**Comment by Ocramius** [comment]
Yes, but who is the target audience, and what is the usage context?

**Comment by malukenho** [comment]
@Ocramius there are no built-in way to encode/decode a php session without change the global state of `$_SESSION` variable. When it's dealing with session handlers implementation, may ya can depare with the necessity of encode/decode the session data or whatever. The audience for sure is small. But I still need it for continue the work on `session handler` using JWT. I just wanted to split it up as I saw in the php manual for example, people trying to implement their on enc/dec for the native php session.

**Comment by Ocramius** [comment]
That doesn't answer the question. I am asking (as a consumer of this library):

 * what am I building?
 * what do I need?
 * why do I need it?
 * how would I use it (example code needed)?

**Comment by necromant2005** [comment]
It makes no sense to support and developing this library, because there's php.ini already has option to change default session storage (if it's required):
session.serialize_handler = php_serialize

Also in case we're talking about PSR7, session must be JWT stored in cookies. So we will be forced to use custom session save handler (if the app requires fallback comparability), so all those cases we control storing and loading session processes.
